core/src/main/resources/schema/app/20220408210524_CDPSDX-3491:_Adding_detach_field_to_stack.sql

```diff
@@ -0,0 +1,9 @@
+-- // CDPSDX-3491: Adding detach field to stack.
+-- Migration SQL that makes the change goes here.
+
+ALTER TABLE stack ADD COLUMN IF NOT EXISTS detached boolean default false;
```

Comment:
I don't think there is meaning to a non-datalake stack that has been detached, no? 

If you're going to add a field either way adding the actual originalName seems more straight forward.